src/main/java/com/bart/scorebetlive442/model/json/datajson/01_createLeague.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Premier League",
+  "country": "England",
+  "level": "Top",
+  "teamIds": [1]
```

Comment:
tutaj właśnie jest przykład klasycznego problemu zależności.
Plik oznaczyłeś jako pierwszy czyli powinien być uruchamiany jako pierwszy w kolejności, tworzymy ligę ale podajemy numer drużyny 1 (której jeszcze nie ma) - wywali się.

Omówimy to na zajęciach jak takie coś programować, pamiętaj że nie wszystko musi być stricte CRUDem, jak mamy ligę możemy np mieć tworzenie ligi (bez drużyn w body) a potem mieć metodę addTeam gdzie podamy tylko numerek ligi i drużyny, return oczywiście może zwracać wszystko i tym się zajmiemy